packages/test/test-version-utils/src/testApi.ts

```diff
@@ -43,6 +43,13 @@ import { TestFluidObjectFactory } from "@fluidframework/test-utils/internal";
 // ContainerRuntime and Data Runtime API
 import * as semver from "semver";
 
+// TypeScript generates incorrect imports in the d.ts file if this is not included.
+import { ISharedObjectKind } from "@fluidframework/shared-object-base/internal";
```

Comment:
Apparent TSC bug workaround.
TODO: follow-up on this (report for find open issue)